### PR TITLE
Update eslintrc to latest version

### PR DESCRIPTION
**Pull request by dancork**
#### What does this PR do? (please provide any background)
The version of `.eslintrc` we were pulling from the culture repo was out-of-date.

#### How can this be tested?
You could pull this branch and npm link from another project

or

You can use the `git+ssh://git@github.com:holidayextras/make-up.git#update-eslintrc` in package.json of another project

#### What gif best describes how you feel about this work?

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by connormeredith** [comment]
Taking review :one:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "make-up",
-  "version": "6.0.0",
+  "version": "6.0.1",
```
Comment (hxpaul) [inline package.json]: what rules does it change? Is it really a patch release?

Comment (pmcnr-hx) [inline package.json]: It needs to be a minor release update if the ruleset changes: `6.1.0`.

Comment (dancork) [inline package.json]: It's the ecmaFeatures for es6. As it's an external dependency and functionality is staying the same it feels like a patch release. Happy to change if you disagree?

Comment (dancork) [inline package.json]: @pmcnr-hx ok, will change :)

**Comment by dancork** [comment]
@pmcnr-hx @hxpaul updated to 6.1.0

**Comment by hxpaul** [comment]
All good here :+1: for review :two:

**Comment by hx-markterry** [comment]
ok with me :hospital:

**Comment by connormeredith** [comment]
Looks good to me :+1:

**Comment by pmcnr-hx** [comment]
:+1:

**Pull request merged**